src/Features/CSharp/Portable/CodeRefactorings/MoveStaticMembers/CSharpMoveStaticMembersCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.MoveStaticMembers;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.MoveStaticMembers
+{
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = "[WIP] Move Members To Type"), Shared]
+    internal class CSharpMoveStaticMembersCodeRefactoringProvider : AbstractMoveStaticMembersRefactoringProvider
+    {
+        /// <summary>
+        /// Test purpose only.
+        /// </summary>
+        [SuppressMessage("RoslynDiagnosticsReliability", "RS0034:Exported parts should have [ImportingConstructor]", Justification = "Used incorrectly by tests")]
+        public CSharpMoveStaticMembersCodeRefactoringProvider(IMoveStaticMembersOptionsService service) : base(service)
```

Comment:
I wonder do we have a new MEF framework to get rid of the test accessor constructor? @sharwell might know more about this.